Bugzilla.pm

```diff
@@ -54,6 +54,7 @@ use File::Basename;
 use File::Spec::Functions;
 use Safe;
 use Sys::Syslog qw(:DEFAULT);
+use JSON::XS qw(decode_json);
```

Comment:
Is BMO generally fine with modules that require compilation? It seems reasonable to me, but in case that's a topic y'all have past decisions on.